spec/helper.cr

```diff
@@ -23,6 +26,12 @@ WebMock.stub(:post, "http://127.0.0.1:2379/v3beta/watch")
   .with(body: "{\"create_request\":{\"key\":\"c2VydmljZS9jb3Jl\",\"range_end\":\"c2VydmljZS9jb3Jm\"}}", headers: {"Content-Type" => "application/json"})
   .to_return(body_io: IO::Stapled.new(*IO.pipe))
 
+WebMock.stub(:get, "http://localhost:6000/api/triggers/v2/version")
+  .to_return(body: "{\"service\":\"#{PlaceOS::Triggers::APP_NAME}\",\"commit\":\"#{PlaceOS::Triggers::BUILD_COMMIT}\",\"version\":\"#{PlaceOS::Triggers::VERSION}\",\"build_time\":\"#{PlaceOS::Triggers::BUILD_TIME}\",\"platform_version\":\"DEV\"}".to_s)
+
+WebMock.stub(:get, "http://localhost:6000/api/triggers/v2/")
+  .to_return(body: "{\"service\":\"#{PlaceOS::Triggers::APP_NAME}\",\"commit\":\"#{PlaceOS::Triggers::BUILD_COMMIT}\",\"version\":\"#{PlaceOS::Triggers::VERSION}\",\"build_time\":\"#{PlaceOS::Triggers::BUILD_TIME}\",\"platform_version\":\"DEV\"}".to_s)
+
```

Comment:
```suggestion
```

Use `with_server`, this isn't actually testing the controller.

Reply:
When I use with_server I get an error: `Real HTTP connections are disabled. Unregistered request: GET http://localhost:6000/api/triggers/v2/version with headers {"Host" => "localhost:6000"}`

Reply:
https://github.com/manastech/webmock.cr

could do `allow_net_connect = true` and then `false` after the specs using `with_server`